.github/workflows/build.yml

```diff
@@ -161,7 +161,7 @@ jobs:
             export CCACHE_DIR=`pwd`/ccache
             mkdir $CCACHE_DIR
             cd sources/testing
-            ./cibuild.sh -c -x testlist/${{matrix.boards}}.dat
+            ./cibuild.sh -c testlist/${{matrix.boards}}.dat
```

Comment:
Please don't forget modify apps and testing git too.

Reply:
Will do.
It is a real pain to maintain this in two places.

Reply:
Yeah two things that will help with this:
1) Moving to GitHub Container Registry from packages (that ticket I shared).  This will mean we get rid of some of the extra logic in the workflow.
2) GitHub has a planned feature to allow sharing workflows between repos.
3) We can move more of the shared logic into a single GitHub action.  This is going to be worth it longer term, just requires motivation to build the action in NodeJS which I have not been excited about.  Maybe I can set a goal for myself to have that in the release after the current one.

Reply:
>     2. GitHub has a planned feature to allow sharing workflows between repos.

That's key